Makefile

```diff
@@ -63,6 +63,26 @@ lint-go:  # lints the Go files
 lint-md:   # lints the Markdown files
 	tools$/prettier$/node_modules$/.bin$/prettier -l .
 
+release:   # creates a new release
+	# cross-compile the binaries
+	goreleaser --rm-dist
+
+	# make Windows installer
+	make --no-print-directory msi
+
+	# create GitHub release with files in alphabetical order
+	hub release create --draft --browse --message v7.4.0 \
+		-a dist/git-town_7.4.0_linux_intel_64.deb \
+		-a dist/git-town_7.4.0_linux_intel_64.rpm \
+		-a dist/git-town_7.4.0_linux_intel_64.tar.gz \
+		-a dist/git-town_7.4.0_linux_arm_64.deb \
+		-a dist/git-town_7.4.0_linux_arm_64.rpm \
+		-a dist/git-town_7.4.0_linux_arm_64.tar.gz \
+		-a dist/git-town_7.4.0_macOS_intel_64.tar.gz \
```

Comment:
Omitting support for BSD. BSD is mostly used on servers these days. I want to see actual need from the community for a BSD release before adding these binaries. I think its better if the list of releases is concise and easy to skim.